aiven/templates/service_user_config_schema.json

```diff
@@ -1256,28 +1700,501 @@
     },
     "type": "object"
   },
+  "kafka_mirrormaker": {
```

Comment:
I would not include this or the m3 related settings as those may still change before making them generally available.